common/config/common.go

```diff
@@ -67,6 +71,7 @@ const (
 const (
 	DefaultAWSRegion           = "us-west-2"
 	DefaultTimeBetweenRequests = time.Millisecond * 10
-	DEFAULT_JOB_EXPIRY         = time.Hour * 1
-	DEFAULT_PORT               = 9090
+	DefaultJobExpiry           = time.Hour * 1
+	DefaultLockExpiry          = time.Hour * 24
```

Comment:
This seems too high. We don't have long running jobs that would take anywhere near a day. Maybe a more pragmatic value of 1 hour or less?

Reply:
This is a good question in that I want to make sure my understanding of the locks being discrete is correct: shouldn't a lock be discretely matched with a job, in which case why would the lock be so long-lived when the job isn't?

Reply:
good points, 1 hour seems to make sense to me